SeriesMaster-Backend/routes/shows.js

```diff
@@ -18,7 +19,13 @@ function createShowsRouter() {
   router.get("/:id", async (req, res) => {
     try {
       const id = req.params.id;
-      const show = await searchSingleShow(id);
+      let show;
+      show = await findShow(database, id);
+
+      if (!show) {
+        show = await searchSingleShow(id);
```

Comment:
I'm not sure if i understand the code right, but with this id, you passing through `searchSingleShow` you get the intended show from the API?

Reply:
Yes, that is correct. Maybe I need to name it more clearly.